docs/deployment/rds-s3/README.md

```diff
@@ -54,95 +75,113 @@ eksctl create cluster \
 ```
 
 4. Create an OIDC provider for your cluster  
-**Important :**  
-You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
+   **Important :**  
+   You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
 
 ## 2.0 Setup RDS, S3 and configure Secrets
-There are two ways to create the RDS and S3 resources before you deploy the Kubeflow manifests. Either use the automated python script we have provided by following the steps in section 2.1 or fall back on the manual setup steps as in section 2.2 below - 
+
+There are two ways to create the RDS and S3 resources before you deploy the Kubeflow manifests. Either use the automated python script we have provided by following the steps in section 2.1 or fall back on the manual setup steps as in section 2.2 below -
```

Comment:
we have a disconnect b/w section 2 and section 3. section 3 talks about how to deploy only rds or s3 but this one does not.

Reply:
currently the auto setup script will setup both RDS and S3 resources, I'll move section 4 above to fill in the discontinuity

Reply:
Yes, we should have the option in script as well. We can leave this script as is in this PR since @ryansteakley is also working on some changes but we need to address this. Maybe we can add this to his PR